CHANGELOG.rst

```diff
@@ -0,0 +1,16 @@
+==================
+frontera changelog
+==================
+
+0.9.0 (2025-01-DD)
+==================
+
+-   Drop support for Python 3.8 and lower, add support for Python 3.9 and
+    higher.
```

Comment:
This seems incomplete, in https://github.com/scrapinghub/scrapy-frontera/pull/14 I have noticed many major, backward-incompatible changes in the main branch that were not present in the last public release.

Reply:
Nevermind, the problem is that scrapy-frontera had frontera frozen at 0.7.1, never got support for 0.8 and 0.8.1, which introduced those breaking changes.